docs/word/citation-management.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Sample: Manage citations through your Word add-in'
+description: Learn how to create an add-in that manages citations in a Word document.
+ms.date: 11/15/2023
+ms.topic: sample
+ms.localizationpriority: medium
+---
+
+# Sample: Manage citations in a Word document using your Word add-in
+
+Citation management is an important aspect of documents, particularly in academia and education. Each citation style has its own guidelines for how citations should be marked in a document as well as where and how the sources should be noted. Examples of citation styles are [APA](http://apastyle.apa.org/) and [MLA](https://www.mla.org/MLA-Style).
+
+This article shows how to create an add-in that showcases the basics of managing citations in a Word document. The add-in displays the references loaded from the user's .bib file then the user can insert selected references into the document.
+
+![Screen showing citation management add-in task pane.](../images/word-citation-management.png)
+
+## Prerequisites
+
+- [Visual Studio Code](https://code.visualstudio.com/Download).
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- npm version 8 or greater.
+- @orcid/bibtexParseJs, a JavaScript library for parsing the selected .bib file.
```

Comment:
Same comment as from the sample: this should be included when they run `npm install`.